### DISCUSSION TITLE
Attachments in v3

### DISCUSSION THREAD

**Discussion by Piotrreek** [xunit/xunit#3063] [Question] [answered]
Is this possible to attach files to test result like it is done in NUnit using TestContext ? 
In NUnit after running the test  I can see Attachments section with attached files in test explorer in VS.

I am using playwright with tracing feature and I would like to attach zip archives to failed tests. Tests are running in Azure Pipelines currently using NUnit, but I would like to switch to XUnit thus I ask this question.

I tried to use XUnit TestContext and attach some file during test run, but I cannot see it in Test Explorer (it’s also not available in Azure DevOps in test summary).

**Comment by bradwilson** [comment, marked as answer]
Added an issue: https://github.com/xunit/visualstudio.xunit/issues/425
